pkgs/servers/blockbook/default.nix

```
@@ -0,0 +1,51 @@
+{ buildGoPackage
+, lib
+, fetchFromGitHub
+, rocksdb
+, bzip2
+, zlib
+, packr
+, snappy
+, pkg-config
+, zeromq
+, lz4
+}:
+
+buildGoPackage rec {
+  pname = "blockbook";
+  version = "0.3.1";
+
+  goPackagePath = "blockbook";
+
+  src = fetchFromGitHub {
+    owner = "trezor";
+    repo = "blockbook";
+    rev = "v${version}";
+    sha256 = "0qgd1f3b4vavw55mvpvwvlya39dx1c3kjsc7n46nn7kpc152jv1l";
+  };
+
+  goDeps = ./deps.nix;
+
+  propagatedBuildInputs = [ bzip2 zlib snappy zeromq lz4 ];
+
+  buildInputs = [ pkg-config packr ];
```

```suggestion
  nativeBuildInputs = [ pkg-config packr ];
```
since they're only needed at build time.